server_process_windows.go

```diff
@@ -4,19 +4,25 @@ package servermanager
 
 import (
 	"context"
-	"fmt"
 	"os"
 	"os/exec"
 )
 
 const ServerExecutablePath = "acServer.exe"
 
-func kill(process *os.Process) error {
-	return exec.Command("TASKKILL", "/T", "/F", "/PID", fmt.Sprintf("%d", process.Pid)).Run()
+func getProcess(cmd *exec.Cmd) *os.Process {
+	return cmd.Process
 }
 
-func buildCommand(ctx context.Context, command string, args ...string) *exec.Cmd {
-	cmd := exec.CommandContext(ctx, command, args...)
+func terminate(ps *os.Process) error {
+	// Windows does not support SIGINT or SIGTERM, just nuke it..
+	return kill(ps)
+}
 
-	return cmd
+func kill(ps *os.Process) error {
+	return ps.Kill()
```

Comment:
`ps.Kill()` doesn't play nice on Windows under some scenarios. From my testing with sTracker and acServer.bat, neither stop correctly, but the main acServer.exe does. As we need to support all three of those, I think the existing (but disgusting) taskkill method may be best. What was the reason for removing the taskkill? I may be missing something here :)

Reply:
That totally makes sense.  I just though maybe the original author wasn't aware of the idiomatic `ps.Kill()` method.  I'll revert that logic to the original technique!